tests/test_prompt.py

```diff
@@ -490,3 +490,67 @@ def test_var_name_value_allowed(
     # Check answers file
     answers = yaml.safe_load((dst / ".copier-answers.yml").read_text())
     assert answers["value"] == "string"
+
+
+@pytest.mark.parametrize(
+    "choices",
+    (
+        [1, 2, 3],
+        [["one", 1], ["two", 2], ["three", 3]],
+        {"one": 1, "two": 2, "three": 3},
+    ),
+)
+def test_list_checkbox(
+    tmp_path_factory: pytest.TempPathFactory,
+    spawn: Spawn,
+    choices: Union[
+        List[int],
+        List[List[Union[str, int]]],  # actually `List[Tuple[str, int]]`
```

Comment:
What does that comment mean?

Reply:
that was just copied from the fixture above, added in https://github.com/copier-org/copier/pull/985 ... 
I presume the intention was to state that it's always going to a list of length two, with a string and an int... but you'd have to ask @sisp why it was put that way

Reply:
A tuple-style choice doesn't actually use a tuple but a list with two elements where the first element is a string and the type of the second element must match the question's type. So semantically the type hint is the type hint for a choice is `Tuple[str, int]`, but since it's a list and not a tuple the actual type is `List[Union[str, int]]` because there is no way to express a tuple that uses a list container.